### DISCUSSION TITLE
Will lazer implement performance optimization for mobile platforms?

### DISCUSSION THREAD

**Discussion by ExHix** [ppy/osu#22525] [Q&A] [answered]
Strong as iPad Pro with M1 cannot run the lazer at 60 fps at all times. Will any performance optimization be implemented for the mobile platforms in the future?

**Comment by mk56-spn** [comment, marked as answer]
Optimisation is an ongoing process, there's a ton of existing discussions on the subject, please search before opening new discussions https://github.com/ppy/osu/discussions?discussions_q=optimisation+mobile
